wherehows-common/src/test/java/wherehows/common/utils/JobsUtilTest.java

```diff
@@ -0,0 +1,104 @@
+/**
```

Comment:
Interesting how git was unable to figure out this is a move?

Reply:
Added two more tests here and half of the lines : )